src/pages/components/externalPages/TestimonialContentTemplate.js

```diff
@@ -0,0 +1,25 @@
+import React, { Fragment } from 'react';
+import TestimonialMainContent from './TestimonialMainContent';
+
+const TestimonialContentTemplate = ({ description }) => {
+
+  return (
+    <Fragment>
+      <header className='flex pt-[64px] sm:pt-[38px] px-[20px] sm:px-[55px] lg:pt-[51px] lg:px-[100px] xl:px-[200px] 2xl:px-[339px]'>
+        <img src='/images/Avatar01.svg' alt='profile' className='h-[71px]' />
```

Comment:
```suggestion
        <LazyLoadImage src='/images/Avatar01.svg' alt='profile' className='h-[71px]' />
```
Change the img tag to LazyLoadImage component
Dont forget also to import the LazyLoadImage